src/olympia/reviewers/tests/test_views.py

```diff
@@ -8150,6 +8171,8 @@ def test_results(self):
             #     versions with needs human review flag
             # - 2 queries for second add-on to get listed/unlisted count of
             #     versions with needs human review flag
+            # - 2 queries for third add-on to get listed/unlisted count of
+            #     versions with needs human review flag
```

Comment:
whaaa?  Does this query count really increase by 2 per add-on?  So what's it like for 100 items?  126 queries?  I'd really recommend a follow-up that prefetches if that's the case.

Reply:
Yeah, because for each add-on, we fetch the number of listed/unlisted. The other scanners queue suffers from the same. Mat noted it might be unavoidable.

Reply:
I don't believe it's unavoidable, but it's probably not easy - it'll take some work and trial and error.  If a simple prefetch on where ever we get the list of records isn't sufficient a custom transformer could work.  And that may lead to changes to the queries in render_addon_name.

Reply:
Yes, it might require some larger changes, because currently the queries are separate and defined in two different places:
* To build the queue, https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/models.py#L331-L342 fetches each add-on (without its versions). 
* To render the add-on name (and the versions), https://github.com/mozilla/addons-server/blob/master/src/olympia/reviewers/utils.py#L252-L268 counts the versions per add-on in the queue